cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -230,6 +232,7 @@ func RunAndMonitor(containerDiskDir, uid string) (int, error) {
 	}
 
 	if pid > 0 {
+		log.Log.Infof("Giving qemu some time to shut down gracefully.")
```

Comment:
I am not sure this is accurate, we are killing the qemu gracefully.

Reply:
Done